metecho/api/models.py

```diff
@@ -313,8 +313,8 @@ class Project(
     github_users = models.JSONField(default=list, blank=True)
     # List of {
     #   "key": str,
-    #   "label": str,
-    #   "description": str,
+    #   "days": int,
+    #   "config_file": str,
```

Comment:
I'm not sure this is helpful. Really this is an object that could have any number of arbitrary properties, but the ones we care about are `key` (required), `label` (optional), and `description` (optional).

Reply:
Oh, I got that from the objects in the staging site and assumed the comment in models.py is outdated. I'll revert 👍